data/i18n/ja/ja.toml

```
@@ -187,6 +193,12 @@ other = "目標"
 [prerequisites_heading]
 other = "始める前に"
 
+[previous_patches]
+other = "パッチリリース"
+
```

I would use an explicit closing paren for `[release_date_after]`
```suggestion

[release_date_after]
other = ")"

```
This protects in case the English locale changes how it works.

Or: we could use `（ ` and `）` (full width parens) instead.